addons/purchase/static/src/js/purchase_datetimepicker.js

```diff
@@ -0,0 +1,3 @@
+$(function () {
+    $('input.datetimepicker-input').datetimepicker();
```

Comment:
shouldn't you make a more precise selector? this will interfere with other portal assets
if this should be generic, it should directly be in base/web module. but let's be secure and select this input in a o_purchase_portal class

Reply:
Yes, I forget to change it.